test/outreach/write/sample_response/prospects.json

```diff
@@ -0,0 +1,443 @@
+{
+  "success": true,
+  "recordId": "9",
+  "data": {
+    "attributes": {
```

Comment:
@laurenzlong should I update the response to include the `attributes` fields as top-level fields in the `data` response?

Reply:
this is good as-is! we want the result of the Write to be the raw response from the API